tests/queries/tests.py

```diff
@@ -3678,3 +3678,11 @@ def test_ticket_24278(self):
         School.objects.create()
         qs = School.objects.filter(Q(pk__in=()) | Q())
         self.assertQuerysetEqual(qs, [])
+
+
+class TestInvalidValuesRelation(TestCase):
+    def test_invalid_values(self):
+        with self.assertRaises(ValueError):
+            Annotation.objects.filter(tag='abc')
```

Comment:
is the exception that's meant to happen `ValueError: invalid literal for int() with base 10: 'abc'`? I thought you might be testing for a nicer error message.